File: 2024_04/0406/middle/anagram.py
count = 0
a1 = [0] * 26
b1 = [0] * 26

a2 = input()
b2 = input()

for c in a2:
    if 'a' <= c <= 'z':
        a1[ord(c) - ord('a')] += 1

for c in b2:
    if 'a' <= c <= 'z':
        b1[ord(c) - ord('a')] += 1

for i in range(26):
    count += abs(a1[i] - b1[i])

print(count)

File: 2024_04/0406/middle/anagram.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int count = 0;
        int[] a1 = new int[26];
        int[] b1 = new int[26];

        String a2 = scanner.next();
        String b2 = scanner.next();

        for (char c : a2.toCharArray()) {
            if (c >= 'a' && c <= 'z') {
                a1[c - 'a']++;
            }
        }

        for (char c : b2.toCharArray()) {
            if (c >= 'a' && c <= 'z') {
                b1[c - 'a']++;
            }
        }

        for (int i = 0; i < 26; i++) {
            count += Math.abs(a1[i] - b1[i]);
        }

        System.out.println(count);
        scanner.close();
    }
}
